### PR TITLE
[7.4 only] LPS-122381

### PR DESCRIPTION
**Pull request by albinojunior**
**Story:** [LPS-105491](https://issues.liferay.com/browse/LPS-105491) | /cc @gabrielpramos @interaminense

### CODE REVIEW

**Comment by albinojunior** [comment]
ci:test:sf

**Comment by albinojunior** [comment]
ci:test:relevant

**Comment by albinojunior** [comment]
ci:stop

File: modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/entry/EditEntry.es.js [hunk 1]
```diff
@@ -87,9 +94,20 @@ export const EditEntry = ({
...
+						fetch(
+							createResourceURL(baseResourceURL, {
+								p_p_resource_id: '/app_builder/add_data_record',
+							}),
+							{
+								body: new URLSearchParams(
+									Liferay.Util.ns(namespace, {
+										appBuilderAppId: appId,
+										dataRecord: JSON.stringify(dataRecord),
+										dataRecordId,
```
Comment (rafaprax) [inline modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/entry/EditEntry.es.js:106]: Hey @albinojunior, we do not need to pass the dataRecordId parameter here. ;)

Comment (albinojunior) [reply modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/entry/EditEntry.es.js:106]: Ok @rafaprax. I have copied the implementation of the app-builder-workflow EditEntry. Should I remove it  from there too?

Thx!

**Comment by albinojunior** [comment]
ci:test:relevant

**Review by gabrielpramos — APPROVED** [review]
LGTM

**Comment by rafaprax** [comment]
ci:forward

**Comment by rafaprax** [comment]
ci:forward

**Pull request closed without merging**